qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|https?://[\w\d-]+\.[\w\d]+"/>
```

Comment:
@JimDeanSpivey according to the original ticket 
> I'd expect all lines containing http:// or https:// to be skipped from this check.

I think  `^import .*$|https?:\/\/.* `  enough in this case
anyway your regex checks only for first two levels in a domain name